examples/scripts/sft_hpu_example.py

```
@@ -0,0 +1,127 @@
+    print(f"Model: {args.model_path}")
+    print(f"Data: {args.data_path}")
+    print(f"Output: {args.ckpt_output_dir}")
+    print(f"GPUs: {args.nproc_per_node}")
+    print(f"Max tokens per GPU: {args.max_tokens_per_gpu:,}")
+    print()
+    
+    # Training configuration optimized for Llama 3.1 8B Instruct
+    start_time = time.time()
+    
+    try:
+        result = sft(
```

Ruff (0.14.1)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)